### ISSUE TITLE
invalid configuration: no configuration has been provided

### ISSUE THREAD

**Issue by Icedroid** [sunny0826/kubecm#9]
```
$ kubecm add -f ~/.kube/local
2020/10/14 16:54:04 invalid configuration: no configuration has been provided

```

**Comment by sunny0826** [comment]
Can you describe it in more detail?

**Comment by Icedroid** [comment]
how to merge new cluster kubeconfig into ~/.kube/config， I try get these error:

kubecm add -f ~/.kube/local
2020/10/14 16:54:04 invalid configuration: no configuration has been provided

**Comment by sunny0826** [comment]
The error is that kubeconfig is not found, can kubectl work?

**Comment by sunny0826** [comment]
fix in #14

**Comment by sunny0826** [comment]
fix is release in [v0.12.0](https://github.com/sunny0826/kubecm/releases/tag/v0.12.0)
